internal/controller/ngrok/kubernetesoperator_controller.go

```diff
@@ -207,6 +208,7 @@ func (r *KubernetesOperatorReconciler) updateStatus(ctx context.Context, ko *ngr
 		ko.Status.RegistrationStatus = ngrokv1alpha1.KubernetesOperatorRegistrationStatusError
 		ko.Status.RegistrationErrorCode = errorCode
 		ko.Status.RegistrationErrorMessage = errorMessage
+		ko.Status.EnabledFeatures = strings.Join(ngrokKo.EnabledFeatures, ",")
```

Comment:
OOC: Why include the enabled features on status as well as spec?

See: https://github.com/ngrok/ngrok-operator/blob/main/api/ngrok/v1alpha1/kubernetesoperator_types.go#L122

Reply:
This is so the additionalPrinterColumn is properly formatted. I ran into https://github.com/kubernetes-sigs/controller-tools/issues/652 and this seems to be the accepted solution on how to accomplish this. I think this also has another benefit in that it will show the true status of the enabled features as seen by ngrok.